src/main/java/de/tudarmstadt/ukp/jwktl/parser/de/components/DEWordFormHandler.java

```diff
@@ -183,9 +188,12 @@ public boolean processBody(final String textLine, final ParsingContext context)
 	protected void extractNounTable(final WiktionaryWordForm wordForm, String label) {
 //		if ("die Koseworte".equals(wordForm.getWordForm()))
 //			System.out.println("X");
-		if (label.endsWith("1") || label.endsWith("2")
-				|| label.endsWith("3") || label.endsWith("4"))
-			label = label.substring(0, label.length() - 1).trim();
+		for (String labelSuffix: NOUN_TABLE_LABEL_SUFFIXES) {
```

Comment:
maybe a RegEx, e.g., `\s\d*\*?\*?$` would be more efficient?

Reply:
I'd leave it like this for now. Most probably I'll rework this for #57 anyway.

Personally, I'm not a fan of regular expressions. I would not be able to understand what `\s\d*\*?\*?$` does without checking the documentation.